### ISSUE TITLE
Consideration: Move all config files to a specific location

### ISSUE THREAD

**Issue by pydanny** [pydanny/cookiecutter-django#439]
On creation, we provide a LOT of configuration files in the root directory. Does it not make sense to put requirements for YUM, APT, various OS tools and more into a specific directory? Perhaps the `requirements` directory?

:bell: @luzfcb @jayfk @theskumar @audreyr

**Comment by jayfk** [comment]
The docker files can't be moved. They have to live on the root of the project because that's the only way to send all relevant files to the docker daemon when building the stack.

The files are:

- Dockerfile
- Dockerfile-dev
- docker-compose.yml
- dev.yml

That being said, I'd vote for a new directory within the project tree. Maybe `utility`? The  `requirements` directory should be reserved for pip installable packages in my opinion. That's makes it easier to distinguish between os dependencies and project dependencies.

**Comment by luzfcb** [comment]
@pydanny @jayfk LGTM. maybe `requirements-os` or `os-requirements` directory? although `utility` is also not bad

**Comment by theskumar** [comment]
I'm more in favour of moving things to `requirements/` or `bin/` directory.

**Comment by Newton715** [comment]
@jayfk @pydanny  The new Version 2 of the docker-compose file enables you to specify a context for the build.  I was able to move all the docker files into a folder.  The Django portion of my compose file looked something like this:
```
  django:
    build:
      context: ../
      dockerfile: docker/Dockerfile-dev
    volumes:
      - ../:/app
```

Nothing had to be edited inside of the Dockerfile.

**Comment by jayfk** [comment]
@Newton715 The problem is that docker-compose will take the new folder as a project name. Looks like `1.7.0` will give us more options in this regard https://github.com/docker/compose/issues/745

**Comment by jayfk** [comment]
I'm closing this since most of this has been fixed in #516
